tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitIntegration.cs

```diff
@@ -85,33 +85,10 @@ internal static class NUnitIntegration
             // Get traits
             if (testMethodProperties != null)
             {
-                var traits = new Dictionary<string, List<string>>();
                 skipReason = (string)testMethodProperties.Get(SkipReasonKey);
-                foreach (var key in testMethodProperties.Keys)
-                {
-                    if (key == SkipReasonKey || key == "_JOINTYPE")
-                    {
-                        continue;
-                    }
-
-                    var value = testMethodProperties[key];
-                    if (value != null)
-                    {
-                        var lstValues = new List<string>();
-                        foreach (var valObj in value)
-                        {
-                            if (valObj is null)
-                            {
-                                continue;
-                            }
-
-                            lstValues.Add(valObj.ToString() ?? string.Empty);
-                        }
-
-                        traits[key] = lstValues;
-                    }
-                }
 
+                var traits = new Dictionary<string, List<string>>();
```

Comment:
tiny optimisation nit: Could make this null initially and use `??=` to initialize inside `ExtractTraits` to improve the case where there's no traits, but probably not worth it

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3777/commits/8d322e285319fad346df08682b6686c25ec868d5